### PR TITLE
File structure

### PR DESCRIPTION
**Pull request by btrettin**
Reworked file structure and removed the old network requests we no longer use. Probably still some changes i need to make so please send feedback.

### CODE REVIEW

**Comment by btrettin** [comment]
youre better off looking at the src itself for the branch rather than the files changed since basically all of them have changed with different imports

**Comment by jd1388** [comment]
What I've noticed that needs changed
* `src/containers/pages/pipeline` should just be `src/containers/pipeline` or `src/containers/connected-pipeline`
* Everything in `src/containers/components` should be in `src/containers`
* Would rather have kebab case for directories and file names
* `src/components/breadcrumb/breadcrumb` contents should be in `src/components/breadcrumb`
* `src/components/login` should be `src/components/login-modal`
* `src/components/navbar/navbar` contents should be in `src/components/navbar`
* `src/components/signup` should be `src/components/signup-model`
* `src/components/errors/index.js` should probably export the `Error404` component as a non default export because in the future, there might be more error components
* `src/pages/homepage/homepage` contents should be in `src/pages/homepage`
* `src/pages/pipeline/requestReview/requestReview` contents should be in `src/pages/pipeline/request-review`
* `src/pages/pipeline/requestSelection/requestSelection` contents should be in `src/pages/pipeline/request-selection`
* `src/reducers/Actions/Actions.js` can live in `src/reducers` and it's directory can be scrapped
* `src/reducers/reducers/` contents can live in `src/reducers`

Need stories for:
* Extracting actions from connected components to their own files (`src/actions`)
* Making sure components are exported in capital camel case
* Extract icons to separate react components
* Find better way to add routes so that we don't end up with one enormous `routes.js` file in the future

This is a good start @btrettin. Those changes are pretty minor. I'll try to add these stories to the backlog sometime tomorrow.

**Comment by btrettin** [comment]
Finished all that you noticed to be changed including changing to kebab. The one thing I didn't change was the default export for error-404, because the linter needs a default export. Is there a way to have the default export not be the component if there is only one? Otherwise everything else was changed. I've also added the other stories so hopefully we can push this and I can start working on those.

**Comment by jd1388** [comment]
@btrettin If you look at the [eslint docs](https://eslint.org/), you should find a way to disable a rule for only one line (you can also search for it on StackOverflow)

**Comment by btrettin** [comment]
I am using  

export { default as Error404_ } from './error-404';   

and that works, but my test has my wrapper as being undefined, so it obviously didn't export it properly. How would you export without a default?

**Comment by jd1388** [comment]
You would import it as
```
import { Error404 } from '.';
```
The curly braces is how you import things that are not the default import.

File: src/components/errors/index.js [hunk 1]
```diff
@@ -1 +1 @@
-export { default } from './error-404'; // eslint-disable-line
+import { Error404 } from '.'; // eslint-disable-line
```
Comment (jd1388) [inline src/components/errors/index.js]: We don't want to disable eslint completely on this line. There's a way to disable the one rule that we need to on this line

Comment (btrettin) [reply src/components/errors/index.js]: Fixed it.

**Pull request merged**